Source/main.py

```diff
@@ -7,8 +7,8 @@
 import os
 import traceback
 import sys
-from pyboy import PyBoy
-from pyboy.logger import addconsolehandler
+from Source.pyboy import PyBoy
```

Comment:
I looks like you are importing the module from the wrong directory. If you need to use PyBoy from another code base, then install the code with `make install`.